python/tvm/contrib/tflite_runtime.py

```diff
@@ -109,3 +110,12 @@ def get_output(self, index):
             The output index
         """
         return self._get_output(index)
+
+    def set_num_threads(self, num_threads):
+        """Set the number of threads via kwargs
+        Parameters
+        ----------
+        num_threads : int
+           The number of threads
+        """
+        self._set_num_threads(num_threads)
```

Comment:
how about adding one new line at the end of file to meet the requirement of PEP8?

Reply:
Makes sense